.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to get an issue fixed
+title: ''
+labels: bug
+assignees: Axel-Naumann
+
+---
+
+**Describe the bug**
+A clear and concise description of what the wrong behavior is.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Your code that triggers the issue: at least a part; ideally something we can run ourselves.
+1. Don't forget to attach the required input files!
+1. How to run your code: build it / `root myMacro.C` / ...
```

Comment:
```suggestion
2. Don't forget to attach the required input files!
3. How to run your code and or build it, e.g. `root myMacro.C`, ...
```
I understand what markdown does, but when the users see the template, the numberings are not markdown-ed yet.